### PR TITLE
Update Send-MsmqQueue.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Missing description for one of the parameter has been added now.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012r2-ps/msmq/Send-MsmqQueue.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ PS C:\> Get-MsmqQueue -Name "FormatName:DIRECT=TCP:10.199.37.61\order_queue" | S
 ## PARAMETERS
 
 ### -AcknowledgeType
-{{Fill AcknowledgeType Description}}
+Specifies the type of acknowledgements supported by Messaging Queue.
```
Comment (get-itips) [inline docset/winserver2012r2-ps/msmq/Send-MsmqQueue.md]:
```suggestion
Specifies the types of acknowledgment message that Message Queuing returns to one or more remote queues
```

Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/msmq/Send-MsmqQueue.md]:
```suggestion
Specifies the type of acknowledgment message that Message Queuing returns to one or more remote queues.
```

**Comment by sowjanya365** [comment]
Incorporated the same changes in the 2016 version as well.

**Pull request merged**